README.md

```diff
@@ -1,14 +1,20 @@
-# Data registry
+# FAIR Data Registry
 
 [![FAIR Data Registry](https://github.com/FAIRDataPipeline/data-registry/actions/workflows/fair-data-registry.yaml/badge.svg)](https://github.com/FAIRDataPipeline/data-registry/actions/workflows/fair-data-registry.yaml) [![codecov](https://codecov.io/gh/FAIRDataPipeline/data-registry/branch/main/graph/badge.svg?token=YT9mHzRfxn)](https://codecov.io/gh/FAIRDataPipeline/data-registry)
 
-The SCRC data registry is a Django website and REST API which is used by the data-pipeline to store metadata about code runs and their inputs and outputs.
+The *FAIR data registry* is a component of the [FAIR data pipeline](https://fairdatapipeline.github.io/) that maintains a metadata catalogue for different data types utilised and generated within a typical epidemiological modelling workflow.
+
+### Implementation 
+
+The *FAIR data registry* is implemented as a [Django](https://www.djangoproject.com/) web application and REST API. The [FAIR data pipeline](https://fairdatapipeline.github.io) uses the data registry to store metadata about code runs and their inputs and outputs. 
+
+### Documentation
 
 User documentation is available [here](docs/index.md).
```

Comment:
I'd move this to the end actually - again it's really for the remote registry, so probably with a comment like the one I added...

```suggestion
```